src/sentry/monitors/endpoints/organization_monitor_index.py

```diff
@@ -187,6 +188,14 @@ def get(self, request: Request, organization: Organization) -> Response:
             sort_fields = ["environment_status_ordering", "-last_checkin_monitorenvironment"]
         elif sort == "name":
             sort_fields = ["name"]
+        elif sort == "owner":
+            queryset = queryset.annotate(
+                owner_ordering=Case(
+                    When(Q(owner_user_id=None) & Q(owner_team_id=None), then=Value(0)),
+                    default=(F("owner_user_id"), F("owner_team_id")),
```

Comment:
Is the idea here that we want to sort owned things above unowned? Should the order actually be based on ids? That'd look fairly random. Maybe we want the default values to be `1` instead, so we show owned first, ordered by name, then unowned, ordered by name

Reply:
oops sorry, this was half done lol

Reply:
I did want to try and sort monitors on the same team / user together

Reply:
That's going to be a bit painful cross-db 😬 At least, it'll be hard to make it look alphabetical

Reply:
this is true 😢